ray_provider/operators/anyscale_job.py

```diff
@@ -0,0 +1,143 @@
+import time
+from typing import Optional, Sequence
+from ray_provider.utils import push_to_xcom
+
+from airflow.utils.context import Context
+from airflow.exceptions import AirflowException
+
+from ray_provider.operators.base import AnyscaleBaseOperator
+from ray_provider.sensors.anyscale_job import AnyscaleProductionJobSensor
+
+from anyscale.shared_anyscale_utils.utils.byod import BYODInfo
+from anyscale.sdk.anyscale_client.models.create_production_job import CreateProductionJob
+
+_POKE_INTERVAL = 60
```

Comment:
What if the user wanted to poll at a different interval maybe a lesser time or more?